examples/DAPDialect/CMakeLists.txt

```diff
@@ -62,3 +62,10 @@ target_link_libraries(buddy-whisper-preprocess
   BuddyLibDAP
   mlir_c_runner_utils
 )
+
+add_executable(buddy-whisper-preprocess-rfft WhisperPreprocessRFFT.cpp)
```

Comment:
please change target `buddy-whisper-preprocess-rfft` to `buddy-rfft`, for it is a standalone operator.